elements/rh-tag/demo/filled-color-with-icon.html

```diff
@@ -1,10 +1,12 @@
-<rh-tag color="red" icon="information">Red</rh-tag>
-<rh-tag color="orange" icon="information">Orange</rh-tag>
-<rh-tag color="green">Green <svg slot="icon" fill="currentColor" viewBox="0 0 512 512" aria-hidden="true" role="img"><path d="M256 8C119.043 8 8 119.083 8 256c0 136.997 111.043 248 248 248s248-111.003 248-248C504 119.083 392.957 8 256 8zm0 110c23.196 0 42 18.804 42 42s-18.804 42-42 42-42-18.804-42-42 18.804-42 42-42zm56 254c0 6.627-5.373 12-12 12h-88c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h12v-64h-12c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h64c6.627 0 12 5.373 12 12v100h12c6.627 0 12 5.373 12 12v24z"></path></svg></rh-tag>
-<rh-tag color="cyan" icon="information">Cyan</rh-tag>
-<rh-tag color="blue" icon="information">Blue</rh-tag>
-<rh-tag color="purple" icon="information">Purple <svg slot="icon" fill="currentColor" viewBox="0 0 512 512" aria-hidden="true" role="img"><path d="M256 8C119.043 8 8 119.083 8 256c0 136.997 111.043 248 248 248s248-111.003 248-248C504 119.083 392.957 8 256 8zm0 110c23.196 0 42 18.804 42 42s-18.804 42-42 42-42-18.804-42-42 18.804-42 42-42zm56 254c0 6.627-5.373 12-12 12h-88c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h12v-64h-12c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h64c6.627 0 12 5.373 12 12v100h12c6.627 0 12 5.373 12 12v24z"></path></svg></rh-tag>
```

Comment:
Are you removing this to bring less attention to the ability to slot a icon as well?  If not should this be broken out into another demo `slotted-icons.html`

Reply:
[c9c3b22](https://github.com/RedHat-UX/red-hat-design-system/pull/1853/commits/c9c3b2201964b16b79e6e0131a10a5bda6a4e76d)